Mathlib/Algebra/Module/Submodule/Pointwise.lean

```diff
@@ -277,4 +289,229 @@ scoped[Pointwise] attribute [instance] Submodule.pointwiseMulActionWithZero
 
 end
 
+section set_acting_on_submodules
```

Comment:
This should have a `/-! ### Subheading -/` and perhaps a few sentences summarizing what this section is about.

Reply:
done in [28d6404](https://github.com/leanprover-community/mathlib4/pull/7140/commits/28d64046205bb588e7dc738321c3bb835505708d)